### DISCUSSION TITLE
Welcome to evm-wallet-and-post-deployment-course Discussions!

### DISCUSSION THREAD

**Discussion by PatrickAlphaC** [Cyfrin/evm-wallet-and-post-deployment-course#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by EngrPips** [comment]
Hello world, I am EngrPips and i am looking to learn and give back to this community

**Comment by Ultra-Rentz** [comment]
Hi Everyone! I am known as Ben and have developed with Solidity for 5 years. However, I am transitioning into Smart Contract Auditing this month. I look forward to taking part in fruitful discussions.

**Reply by EngrPips** [reply]
Welcome Aboard.
